libs/movie/src/lib/movie/form/media-notes/notes.component.ts

```diff
@@ -47,4 +50,10 @@ export class MovieFormMediaNotesComponent implements OnInit, OnDestroy {
   ngOnDestroy() {
     this.sub.unsubscribe();
   }
+
+  removeFromQueue(index: number) {
+    const storagePath = getFileStoragePath('movies', 'notes', this.movieId);
+    const queueIndex = getFileListIndex(index, this.form.promotional.get('notes').value);
+    this.uploaderService.removeFromQueue(storagePath, queueIndex);
+  }
```

Comment:
Prefer using the value directly instead of mixing `getter`, `get()` & `value`
```suggestion
  removeFromQueue(index: number) {
    const title = this.form.value;
    const storagePath = getFileStoragePath('movies', 'notes', this.movieId);
    const queueIndex = getFileListIndex(index, title.promotional.notes);
    this.uploaderService.removeFromQueue(storagePath, queueIndex);
  }
```